recipes/perl-text-fuzzy/LICENSE

```diff
@@ -0,0 +1,127 @@
+			 The "Artistic License"
```

Comment:
Since it is an either/or license, to be complete you may want to include the GPL 1.0 text too. The exact requirements here are a bit unclear to me though.

Reply:
Since it's unclear and I might never know what the actual requirement is until someone merges, I'll leave it as-is with a single license file.